discord/ext/commands/bot.py

```diff
@@ -954,7 +954,7 @@ class Bot(BotBase, discord.Client):
 
     Attributes
     -----------
-    command_prefix
+    command_prefix: Union[:class:`str`, Iterable[:class:`str`], Callable[[:class:`Bot`, :class:`.Message`], Union[:class:`str`, Iterable[:class:`str`]]]]
```

Comment:
```suggestion
    command_prefix
```

For things where the typing are too complicated I just leave them out.